app/gui/src/presenter/project.rs

```diff
@@ -416,6 +426,8 @@ impl Project {
             view.set_read_only <+ view.toggle_read_only.map(f_!(model.toggle_read_only()));
             eval graph_view.execution_environment((env) model.execution_environment_changed(*env));
             eval_ graph_view.execution_environment_play_button_pressed( model.trigger_clean_live_execution());
+
+            eval_ view.dashboard_button_pressed (model.close_ide());
```

Comment:
I find the name `dashboard_button_pressed` confusing, as there are multiple buttons in the dashboard. Which dashboard button does this code refers to?

Reply:
This is in the project view, so the only thing it could refer to is the button to go to the dashboard. If there's a name that makes more sense to you, feel free to suggest it.